cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -61,12 +61,4 @@ spec:
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
-        livenessProbe:
```

Comment:
yeah, but the manifest is materially different so we want a version on it for now.